contribute/developer-guide.md

```diff
@@ -274,23 +277,23 @@ To build a Docker image, run:
 make build-docker-full
 ```
 
-The resulting image will be tagged as grafana/grafana:dev.
+The resulting image will be tagged as `grafana/grafana:dev`.
 
-**Note:** If you are using Docker for macOS, be sure to set the memory limit to be larger than 2 GiB. Otherwise, `grunt build` may fail. The memory limit settings are available under **Docker Desktop** -> **Preferences** -> **Advanced**.
+> **Note:** If you use Docker for macOS, be sure to set the memory limit to be larger than 2 GiB. Otherwise, `grunt build` may fail. The memory limit settings are available under **Docker Desktop** -> **Preferences** -> **Advanced**.
```

Comment:
Another one that could be an admonition?  I see several others in other text that wasn't touched in this edit pass as well.